client/src/components/SendMessage.js

```diff
@@ -36,36 +39,73 @@ class SendMessage extends React.Component {
     return;
   }
 
+  hideEmojiPicker() {
+      this.setState({
+          ...this.state,
```

Comment:
In general, you don't need to pass the current state in React when changing the state. React is clever in a way that it combines the passed parameters in setState with the current state.